include/openssl/obj_mac.h

```diff
@@ -1583,9 +1583,10 @@
 #define NID_qcStatements                286
 #define OBJ_qcStatements                OBJ_id_pe,3L
 
-#define SN_ac_auditEntity               "ac-auditEntity"
-#define NID_ac_auditEntity              287
-#define OBJ_ac_auditEntity              OBJ_id_pe,4L
+#define SN_ac_auditIdentity             "ac-auditIdentity"
+#define LN_ac_auditIdentity             "X509v3 Audit Identity"
+#define NID_ac_auditIdentity            1321
+#define OBJ_ac_auditIdentity            OBJ_id_pe,4L
```

Comment:
This is an API and ABI break. Could this be changed so the NID_ac_auditIdentity keeps the value 287 and the removed defines be made aliases of the new defines in the objects.h file?
It would still be a slight break but IMO an acceptable one.

Reply:
As I said above that would still be an ABI break but IMO acceptable one.

Reply:
Resolved.